DigitalLearningSolutions.Data/DataServices/UserDataService.cs

```diff
@@ -136,6 +137,49 @@ public UserDataService(IDbConnection connection)
             return user;
         }
 
+        public DelegateUserCard? GetDelegateUserCardById(int id)
+        {
+            var user = connection.Query<DelegateUserCard>(
+                @"SELECT
+                        cd.CandidateID AS Id,
+                        cd.CandidateNumber,
+                        ct.CentreName,
+                        cd.CentreID,
+                        cd.DateRegistered,
+                        ct.Active AS CentreActive,
+                        cd.EmailAddress,
+                        cd.FirstName,
+                        cd.LastName,
+                        cd.Password,
+                        cd.Approved,
+                        cd.Answer1,
+                        cd.Answer2,
+                        cd.Answer3,
+                        cd.Answer4,
+                        cd.Answer5,
+                        cd.Answer6,
+                        cd.JobGroupId,
+                        jg.JobGroupName,
+                        cd.SelfReg,
+                        cd.ExternalReg,
+                        cd.Active,
+                        (SELECT AdminID
+                         FROM AdminUsers au
+                         WHERE au.Email = cd.EmailAddress
+                         AND au.CentreID = cd.CentreID
+                         AND au.Email != ''
+                        ) AS AdminID,

```

Comment:
I can't remember the exact logic for matching accounts, but I think it might go beyond having the same email address and it could be that the Alias matches the admin user email address BUT also, the password hash has to match. Do you mind checking the logic under "GetVerifiedLinkedAccounts" or similar method in UserService and updating the docs: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/19319816280/User+Accounts+Admins+and+Delegates+Candidates

Happy to help you dig into it if you can't find the method (as the name came off the top of my head right now)